bundle/schema/docs/bundle_descriptions.json

```diff
@@ -1472,7 +1472,6 @@
                     "config": {
                       "description": "The model serving endpoint configuration.",
                       "properties": {
-                        "description": "",
                         "properties": {
```

Comment:
Note: model serving docs are missing from the generated schema because the schema format here is incorrect. All this means is that the docs are not loaded, but the schema command still works. I have filed a internal followup to fix these so the docs are loaded successfully.

For now, the description field was removed by hand.